modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/resources/com/liferay/dynamic/data/mapping/util/packageinfo

```diff
@@ -1 +1 @@
-version 2.8.0
\ No newline at end of file
+version 3.0.0
```

Comment:
I share [your doubts](https://github.com/liferay-frontend/liferay-portal/pull/730#issuecomment-768985648) about this, but looking on `master`, I believe that this got updated already in 1afa477bcdb73.

Can you please rebase to drop this?